src/test/java/com/bringframework/integration/IntegrationTest.java

```diff
@@ -0,0 +1,30 @@
+package com.bringframework.integration;
+
+import com.bringframework.BringApplication;
+import com.bringframework.context.ApplicationContext;
+import com.bringframework.integration.service.ChatService;
+import com.bringframework.integration.service.GreetingService;
+import com.bringframework.integration.service.IMessageService;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class IntegrationTest {
```

Comment:
Integration tests by convention have postfix `IT`. 
Lets rename this test class to  `BringApplicationIT`